src/Microsoft.AspNetCore.Blazor.Razor.Extensions/BlazorDiagnosticFactory.cs

```diff
@@ -30,23 +30,14 @@ public static RazorDiagnostic Create_CodeBlockInAttribute(SourceSpan? source, st
             return diagnostic;
         }
 
-        public static readonly RazorDiagnosticDescriptor ExpressionInAttributeList =
-            new RazorDiagnosticDescriptor(
+        public static readonly RazorDiagnosticDescriptor UnclosedTag = new RazorDiagnosticDescriptor(
             "BL9980",
-            () =>
-                "Expressions like '{0}' inside of a tag must be part of an attribute. " +
-                "Previous releases of Blazor supported constructs like '@onclick(...)' or '@bind(...)'." +
-                "These features have been changed to use attribute syntax. " +
-                "Use 'onclick=\"@...\"' or 'bind=\"...\" respectively.",
```

Comment:
I removed this error. We had discussed removing it in 0.3 previously, and it would have required some work to retain.